conans/model/scm.py

```diff
@@ -16,6 +16,13 @@ def get_scm_data(conanfile):
 
 class SCMData(object):
 
+    @staticmethod
+    def _boolean(value, default):
+        if value is None:
+            return default
+        else:
+            return "True" if value in [True, "True"] else "False"
```

Comment:
Shall I protect against _invalid_ values? I mean, check that the value is `True, "True", False, "False"`?